app/src/main/java/com/twilio/video/app/sdk/RoomManager.kt

```diff
@@ -59,6 +47,8 @@ class RoomManager(
 
     fun disconnect() {
         room?.disconnect()
+        localParticipantManager.removeAudioTrack()
+        localParticipantManager.removeCameraTrack()
```

Comment:
I'm trying to understand the bug better. So after a user kills the app, the participant stays in the room forever and both users can still see and hear each other? Well, I guess the user that killed the app would not be able to see the other user because there is no UI.

Reply:
According to my new understanding about the disconnection process, I will remove these two lines. We don't need to call this as long as the method `room.disconnect()` is also called.

Reply:
👍